docs/basics/data-format.md

```diff
@@ -4,15 +4,17 @@
 Finetuner uses Jina [`Document`](https://docs.jina.ai/fundamentals/document/) as the primitive data type. In
 particular, [`DocumentArray`](https://docs.jina.ai/fundamentals/document/documentarray-api/)
 and [`DocumentArrayMemap`](https://docs.jina.ai/fundamentals/document/documentarraymemmap-api/) are the input data type
-in the high-level `finetuner.fit()` API. This means, your training dataset and evaluation dataset should be stored in `DocumentArray`
-or `DocumentArrayMemap`, where each training or evaluation instance is a `Document` object:
+in the high-level `finetuner.fit()` API. This means, your training dataset and your validation data (query/index dataset)
```

Comment:
to me is very hard to understand what is expected in these `query` and `index` data.

Reply:
yeah we should explain it better in the docs